editor/src/components/canvas/controls/yoga-control.tsx

```diff
@@ -67,11 +69,44 @@ class YogaResizeControl extends React.Component<YogaResizeControlProps> {
       // TODO check me
       return null
     }
-
+    let labels: {
+      vertical: 'flexBasis' | 'FlexCrossBasis'
+      horizontal: 'flexBasis' | 'FlexCrossBasis'
+    } = {
+      vertical: 'flexBasis',
+      horizontal: 'FlexCrossBasis',
+    }
+    const parentPath = EP.parentPath(this.props.target)
+    const parentElement = withUnderlyingTarget(
+      parentPath,
+      this.props.projectContents,
+      this.props.nodeModules,
+      this.props.openFile,
+      null,
+      (_, element) => {
+        return element
+      },
+    )
+    if (parentElement != null) {
+      const flexDirection = eitherToMaybe(FlexLayoutHelpers.getMainAxis(right(parentElement.props)))
+      if (flexDirection === 'vertical') {
+        // column, column-reverse
+        labels = {
+          horizontal: 'FlexCrossBasis',
```

Comment:
this should be width or height based on the flex direction, because flex cross basis was removed with the pragma

Reply:
There's still a whole lot of code that still refers to `FlexCrossBasis`. :/

Reply:
Fixed in e70865c343b6165cba36aff00f57c14962a961ea.